config/initializers/productization.rb

```diff
@@ -1 +1,3 @@
-Vmdb::Productization.new.prepare
+Rails.application.config.to_prepare do
```

Comment:
Tried the suggestions by @Fryguy 

Doing it in `after_initialize` didn't work, but doing it in `initializer` with `append_asset_paths` seems to work, at least for the assets.paths.  Don't know much about the assets, and can't say this verifies this approach is good or not, though.